frontend/src/main/components/BasicCourseSearch/CourseOverTimeInstructorSearchForm.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from "react";
+import { Form, Button, Container, Row, Col } from "react-bootstrap";
+
+//import { allTheLevels } from "fixtures/levelsFixtures";
+import { quarterRange } from "main/utils/quarterUtilities";
+
+import { useSystemInfo } from "main/utils/systemInfo";
+import SingleQuarterDropdown from "../Quarters/SingleQuarterDropdown";
+// import SingleSubjectDropdown from "../Subjects/SingleSubjectDropdown";
+//import SingleLevelDropdown from "../Levels/SingleLevelDropdown";
+// import { useBackend  } from "main/utils/useBackend";
+
+const CourseOverTimeInstructorSearchForm = ({ fetchJSON }) => {
+
+  const { data: systemInfo } = useSystemInfo();
+
+  // Stryker disable OptionalChaining
+  const startQtr = systemInfo?.startQtrYYYYQ || "20211";
+  const endQtr = systemInfo?.endQtrYYYYQ || "20214";
+  // Stryker restore OptionalChaining
+
+  const quarters = quarterRange(startQtr, endQtr);
+
+  // Stryker disable all : not sure how to test/mock local storage
+  const localStartQuarter = localStorage.getItem("CourseOverTimeInstructorSearch.StartQuarter");
+  const localEndQuarter = localStorage.getItem("CourseOverTimeInstructorSearch.EndQuarter");
+  const localInstructor = localStorage.getItem("CourseOverTimeInstructorSearch.Instructor");
+
```

Comment:
We need  this to match with the `// Stryker disable all`
```suggestion
// Stryker restore all
```